libs/src/services/creatorNode/index.js

```diff
@@ -73,6 +73,25 @@ class CreatorNode {
     return null
   }
```

Comment:
this file will also disappear in diff when this pr is merged https://github.com/AudiusProject/audius-protocol/pull/1218